scripts/zones/Mhaura/npcs/Orlando.lua

```diff
@@ -40,5 +88,22 @@ end;
 function onEventFinish(player,csid,option)
 	-- printf("CSID: %u",csid);
 	-- printf("RESULT: %u",option);
-end;
+    local QuestStatus = player:getQuestStatus(OTHER_AREAS, ORLANDO_S_ANTIQUES);
+    local payout = player:getVar("ANTIQUE_PAYOUT");
 
+    if (csid == 0x0065) then
+        player:addQuest(OTHER_AREAS, ORLANDO_S_ANTIQUES);
+    elseif (csid == 0x0066) then
+        player:tradeComplete();
+        player:addFame(WINDURST,WIN_FAME*10);
+        player:addGil(payout);
+        player:messageSpecial(GIL_OBTAINED,payout);
+        player:completeQuest(OTHER_AREAS, ORLANDO_S_ANTIQUES);
+        player:setVar("ANTIQUE_PAYOUT", 0);
+        player:setLocalVar("OrlandoRepeat", 0);
+    elseif (csid == 0x0067) then
+        if (QuestStatus == QUEST_COMPLETED) then
+            player:setLocalVar("OrlandoRepeat", 1);
```

Comment:
@xdemolish @waytim You guys...You can still `addQuest` a quest that has been repeated, last I knew. And payout would have worked with get/setLocalvar's for it since its created mere seconds before being used and then zero'd back out. :(